docs/howto/manage-configurations.md

```diff
@@ -69,56 +68,27 @@ https://github.com/canonical/juju-sdk-tutorial-k8s/compare/01_create_minimal_cha
 
 > See first: {ref}`get-started-with-charm-testing`
 
-You'll want to add two levels of tests: unit and scenario.
-
 ### Write unit tests
 
 > See first: {ref}`write-unit-tests-for-a-charm`
 
-To use a unit test to verify that the configuration change is handled correct, the test needs to trigger the `config-changed` event and then check that the update method was called. In your `tests/unit/test_charm.py` file, add the following test functions to the file:
+To verify that the `config-changed` event validates the port, pass the new config to the `State`, and, after running the event, check the unit status. For example, in your `tests/unit/test_charm.py` file, add the following test function:
 
 ```python
-def test_invalid_port_configuration():
-    harness = ops.testing.Harness()
-    harness.begin()
-
-    harness.update_config({"server-port": 22})
-    assert isinstance(harness.model.unit.status, ops.BlockedStatus)
-
-def test_port_configuration(monkeypatch):
-    update_called = False
-    def mock_update(*args):
-        update_called = True
-    monkeypatch.setattr(MyCharm, "_update_layer_and_restart", mock_update)
-
-    harness = ops.testing.Harness()
-    harness.begin()
-
-    harness.update_config({"server-port": 8080})
+from ops import testing
 
-    assert update_called
-```
-
-### Write scenario tests
-
-> See first: {ref}`write-scenario-tests-for-a-charm`
-
-To use a Scenario test to verify that the `config-changed` event validates the port, pass the new config to the `State`, and, after running the event, check the unit status. For example, in your `tests/scenario/test_charm.py` file, add the following test function:
-
-```python
 def test_open_port():
-    ctx = scenario.Context(MyCharm)
+    ctx = testing.Context(MyCharm)
 
-    state_out = ctx.run("config_changed", scenario.State(config={"server-port": 22}))
+    state_out = ctx.run(ctx.on.config_changed(), testing.State(config={"server-port": 22}))
 
-    assert isinstance(state_out.unit_status, ops.BlockedStatus)
+    assert isinstance(state_out.unit_status, testingZ.BlockedStatus)
 ```
 
-### Test-deploy
+### Manually test
 
```

Comment:
Good catch. (When I chose 'Test-deploy' I was just looking for a way to file the `juju deploy` test under the testing section, but of course it's not good enough because integration testing also test-deploys.)